tests/Generator/CollisionHandlerTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Ahc\Phint\Test;
+
+use Ahc\Phint\Generator\CollisionHandler;
+use PHPUnit\Framework\TestCase;
+
+class CollisionHandlerTest extends TestCase
+{
+    public function testConstructor()
+    {
+        $this->assertInstanceOf(CollisionHandler::class, new CollisionHandler());
+    }
+
+    public function testHandle()
+    {
+        $targetjsonFile   = __DIR__ . '/../fixtures/example.json';
+        $collisionHandler = new CollisionHandler();
+        $collisionHandler->handle($targetjsonFile, \json_encode(['key' => 'value']));
```

Comment:
should we have two keys: one for merging, other for totally new value?

should we also test `.md` collision (append) in different `testMethod()`?

Reply:
I think the `.md` file test is in the next PR to do that.